app/assets/stylesheets/pages.css.scss

```diff
@@ -144,6 +144,25 @@
   margin-bottom: 10px;
 }
 
+.content .edit_link {
+  font-size: 60%;
+  padding: 0;
+  text-align: right;
+  width: 100%;
+
+  a {
+    border: solid 1px lightgray;
```

Comment:
Color `lightgray` should be written in hexadecimal form as `#d3d3d3`<br>Color literals like `lightgray` should only be used in variable declarations; they should be referred to via variable everywhere else.